container_registry/container_analysis/poll_discovery_finished.go

```diff
@@ -52,8 +52,19 @@ func pollDiscoveryOccurrenceFinished(resourceURL, projectID string, timeout time
 		case <-ticker.C:
 			req := &grafeaspb.ListOccurrencesRequest{
 				Parent: fmt.Sprintf("projects/%s", projectID),
-				Filter: fmt.Sprintf(`kind="DISCOVERY" AND resourceUrl=%q`, resourceURL),
+				// Vulbnerability discovery occurrences are always published under the
+				// PACKAGE_VULNERABILITY note in the "goog-analysis" GCP project.
```

Comment:
Not sure I understand this - if the project is locked down, should it be in this example?

Reply:
Whenever a user pushes an image to GCR with vulnerability scanning enabled, Google will scan the image and provide details about any vulnerabilities found in a publicly readable GCP project.  This sample is showing how to query that project to find the results of the vulnerability scanning, which is a common use case. 

The issue comes when we want to write to the project for testing, which is why I added the fall back filter